dream_demo/bin/form.eml.html

```diff
@@ -0,0 +1,21 @@
+let render request attrs selected_dog_option =
```

Comment:
Same comment about the auto-formatting -- it obscures the actual change, sorry about that.